honeybee/_hbanalysissurface.py

```diff
@@ -78,16 +82,32 @@ def from_json(cls, srf_json):
         """Create a surface from json object.
 
         The minimum schema is:
-        {"name": "",
-        "vertices": [[(x, y, z), (x1, y1, z1), (x2, y2, z2)]],
-        "surface_type": null  // 0: wall, 5: window
-        }
+            {"name": "",
+            "vertices": [[(x, y, z), (x1, y1, z1), (x2, y2, z2)]],
+            "surface_material": {},  // radiance material json file
+             "surface_type": null  // 0: wall, 5: window
+            }
         """
         name = srf_json["name"]
         vertices = srf_json["vertices"]
         type_id = srf_json["surface_type"]
         srf_type = surfacetype.SurfaceTypes.get_type_by_key(type_id)
-        return cls(name, vertices, srf_type)
+        HBsrf = cls(name, vertices, srf_type)
+        # Check material type and determine appropriate "from_json" classmethod
+        if "surface_material" in srf_json.keys():
```

Comment:
`if "surface_material" in srf_json` will be just as good and probably more "Pythonic"!